src/components/map/OlMap.vue

```diff
@@ -27,6 +27,14 @@
                                 {{ layer.get('title') }}
                             </div>
                         </td>
+                        <td v-if="showProtectionAreas && !editable">
```

Comment:
Ergo : this check box and its title should be probably be placed above slopes part : try to the map on a route on a medium size screen : it is not visible as now.

![capture d ecran 2019-02-10 a 10 35 42](https://user-images.githubusercontent.com/11915659/52532092-ab23d880-2d1f-11e9-86aa-1b2b04639a5b.png)

Reply:
Yes, one of the L&F problems, not dealt yet